src/dnx.mono.managed/EntryPoint.cs

```diff
@@ -89,12 +89,28 @@ private static string[] ExpandCommandLineArguments(string[] arguments)
             return arguments;
         }
 
+        var argExpanded = false;
         var expandedArgs = new List<string>();
         for (var i = 0; i < arguments.Count(); i++)
         {
-            if (i == parameterIdx)
+            if (!argExpanded)
             {
-                ExpandArgument(arguments[i], expandedArgs);
+                if (arguments[i] == "-p" || arguments[i] == "--project")
```

Comment:
No, this should be aligned with what we already have. I will fix this to be case insensitive.